ts-eks/test/klotho.yaml

```diff
@@ -5,3 +5,32 @@ out_dir: compiled
 defaults:
     execution_unit:
         type: eks
+execution_units:
+    users-api:
+        network_placement: private
+        pulumi_params:
+            nodeType: node
+            replicas: 1
+            nodeConstraints:
+                instanceType: c4.large
+                diskSize: 20
+            limits: 
+                cpu: 1
+                memory: 256
+            autoscalingConfig:
+                cpuUtilization: 100
+                memoryUtilization: 100
+                maxReplicas: 4
+    users-store:
+        network_placement: public
+        pulumi_params:
+            nodeType: node
+            nodeConstraints:
+                instanceType: t3.large
+                diskSize: 200
+    user-info:
+        network_placement: private
+        pulumi_params:
+            nodeType: node
+            nodeConstraints:
+                diskSize: 100
```

Comment:
does this also need instance type? i dont think there is a default instance type set right, so it'll just be ignored in the checks in ur pulumi test.

Reply:
it will get bundled with the c4.large. We do default to t3.medium if theres no node group that we can combine it with though